config/grain.json

```diff
@@ -1,5 +1,11 @@
 {
   "allocationPolicies": [
+    {
+      "budget": 0.25,
```

Comment:
Is this the budget for the total week of SourceCred rewards? Or per user per week?

Reply:
I was reading SourceCred's documentation, I interpret this as weekly totals, correct me if I'm wrong

Reply:
> Is this the budget for the total week of SourceCred rewards? Or per user per week?

@luisivan Yes, this is the total weekly budget. So about 1.07 `$NATION` per month, to reward contributions in Discourse/Discord/GitHub.